modules/ROOT/pages/to-use-api-notebook.adoc

```diff
@@ -91,7 +91,14 @@ API.authenticate(github);
 ----
 ====
 
+== API Notebook security
+
+API notebooks run plain JavaScript in the browser. To ensure sensitive data does not leak, API Notebook disables certain JavaScript functionality:
+
```

Comment:
You don't need to end the following list items with periods, because they are both fragments.

Reply:
I'm a little embarrassed to have missed this, and to have missed the capitalization. Thank you very much for fixing them both!